src/lambdas/ingest/index.js

```diff
@@ -4,6 +4,22 @@ import { createIndex } from '../../lib/database-client.js'
 import { processMessages, publishResultsToSns } from '../../lib/ingest.js'
 import getObjectJson from '../../lib/s3-utils.js'
 import logger from '../../lib/logger.js'
+import { AssetProxy } from '../../lib/asset-proxy.js'
+
+let assetProxyInstance = null
+
+const getAssetProxy = async () => {
+  if (!assetProxyInstance) {
+    assetProxyInstance = new AssetProxy()
+    await assetProxyInstance.initialize()
+  }
```

Comment:
Again, same comment: initializing this inside the function call means we don't initialize during the lambda init phase.

Reply:
Updated.